app/src/main/java/com/OxGames/OxShell/Wallpaper/Shader.java

```diff
@@ -325,15 +325,16 @@ protected void draw() {
 
         frame++;
         long currentTime = System.currentTimeMillis() - startTime;
-        float secondsElapsed = (currentTime) / 1000f; // will be input into shader as iTime
-        float deltaTime = (currentTime - prevTime) / 1000f;
-        float fps = 1 / deltaTime;
+        double secondsElapsed = (currentTime) / 1000.0; // will be input into shader as iTime
+        double deltaTime = (currentTime - prevTime) / 1000.0;
+        double fps = 1 / deltaTime;
```

Comment:
Changing to double causes issue when passing the parameter to the shader, please switch back to float